src/server.c

```diff
@@ -749,7 +749,7 @@ struct redisCommand redisCommandTable[] = {
      0,NULL,0,0,0,0,0,0},
 
     {"exec",execCommand,1,
-     "no-script no-monitor no-slowlog ok-loading ok-stale @transaction",
+     "no-script no-slowlog ok-loading ok-stale @transaction",
```

Comment:
I think we should also remove `no-monitor` and `no-slowlog` from `HELLO` and `AUTH` command.

And use `redactClientCommandArgument` in authCommand.

Reply:
Alright, also verified the performance drop was actually really small compared to the SHA execution.